metron-deployment/packaging/docker/rpm-docker/SPECS/metron.spec

```diff
@@ -274,6 +276,23 @@ This package installs the Metron Indexing files
 %{metron_home}/config/zeppelin/metron/metron-ip-report.json
 %{metron_home}/config/zeppelin/metron/metron-connection-volume-report.json
 
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  
```

Comment:
Even though it doesn't functionally matter maybe we should trim off those squares?